scene/console.lua

```diff
@@ -334,7 +335,7 @@ local commands={} do
         details={
             "Show a message on the up-left corner",
             "",
-            "Usage: mes <check|info|warn|error>",
+            "Usage: mes <check|info|warn|error|other>",
```

Comment:
there is no 'other' option now

Reply:
An `other` option is actually present in this command's code and in-command help message when you provide an invalid argument. It actually works in-game (gives a gray message display without an icon).